src/Features/VisualBasic/Portable/Completion/CompletionProviders/ImplementsClauseCompletionProvider.vb

```diff
@@ -171,8 +171,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
             Next
 
             Dim symbols = semanticModel.LookupSymbols(position)
+            Dim availableInterfacesAndContainers = interfacesAndContainers.Where(Function(s) symbols.Contains(s.OriginalDefinition)).ToImmutableArray()
```

Comment:
❗️ 

The following operations are both O(n)

* `interfacesAndContainers.Where(...).ToImmutableArray()`
* `symbols.Contains(s.OriginalDefinition)`

These operations are nested, making the overall operation O(n²).

However, the original expression used a sequence (as opposed to nesting) of O(n) operations. This is a regression in the algorithmic complexity of the completion provider.